docs/specs/markdown.yml

```diff
@@ -608,3 +608,35 @@ outputs:
                           <a href=\"a#\" data-linktype=\"relative-path\">a1</a>
                      </p>"
     }
+---
+# Triple colon markdown extension
+inputs:
+  docfx.yml:
+  docs/a.png:
+  docs/b.png:
+  docs/a.md: |
+    :::image type="content" source="a.png" alt-text="Image":::
+    :::image type="content" source="~/docs/b.png" alt-text="Image":::
+outputs:
+  docs/a.png:
+  docs/b.png:
+  docs/a.json: |
+    { 
+      "conceptual": "<img src=\"a.png\" alt=\"Image\" data-linktype=\"relative-path\"><img src=\"b.png\" alt=\"Image\" data-linktype=\"relative-path\">",
+    }
+  .links.json: |
```

Comment:
nit: consider remove the verification of `.links.json` and `.dependencymap.json` to simplify the test case. Verifying output content is probably enough to show that links in triple colon images are resolved as other links. Other resolve tests cases are also written without checking these files. There are dedicated test cases to check `.links.json` and `.dependencymap.json` generation.

Reply:
make sense, I will move the .link.json validation to the specific case